docs/detections/session-view.asciidoc

```diff
@@ -84,4 +89,23 @@ Session View includes an additional badge not pictured above:
 * The *Exec user change* badge highlights exec user changes, such as when a user escalates to root:
 +
 [role="screenshot"]
-image::images/session-view-exec-user-change-badge.png[The Exec user change button,width=80%,height=80%]
+image::images/session-view-exec-user-change-badge.png[The Exec user change badge,width=80%,height=80%]
+
+* The *Output* badge appears next to commands which generated TTY output. Click it to open Terminal output view.
++
+[role="screenshot"]
+image::images/session-view-output-badge.png[The Output badge,width=80%,height=30%]
+
+[[session-view-output]]
+[discrete]
+=== Terminal output view UI
+Terminal output view can help provide a more complete understanding of commands executed by users or adversaries, and assist with auditing and compliance objectives. It allows you to review a session's TTY output, in the context of the commands which generated it. It includes both TTY output, and commands entered into a shell as typed — provided the shell was in echo mode (for background information about Linux terminals, refer to https://github.com/elastic/security-team/blob/main/docs/adaptive-workload-protection-team/architecture/linux-terminals.md[Linux terminals]).
+
+[role="screenshot"]
+image::images/session-view-output-viewer.png[Terminal output view,width=100%,height=100%]
+
+1. The search bar. Use it to find and highlight search terms within the current session.
+The buttons on its right side allow you to jump through search results.
+2. The right-side scroll bar. Use it along with the bottom-side scroll bar to navigate output data that doesn't fit on a single screen.
+3. The playback controls: *Start*, *Prev*, *Play*, *Next*, and *Last*. Use them to advance or rewind the session's commands and output.
+4. The *Fit screen*, *Zoom in*, and *Zoom out* buttons. Use them to adjust the text size.
```

Comment:
might be worth noting that using the existing "fullscreen" toggle is advised when viewing sessions whose host window size is large. In order to accurately render terminal output for graphical programs like vim, the sessions tty row and column configuration is captured. You can see this displayed in the top right corner of the terminal window boundary. A users terminal can also be resized at any point during a session, so this will be reflected when playing back terminal output.